### ISSUE TITLE
log-level help

### ISSUE THREAD

**Issue by MicahZoltu** [docker/docker#26888]
**Description**

Docker help and docker reference docs do not give details on how to use the `--log-level` command.

**Steps to reproduce the issue:**
1. docker help
2. https://docs.docker.com/engine/reference/commandline/cli/

**Describe the results you received:**
No help on how to set `--log-level` to anything other than `info`.

**Describe the results you expected:**
Other log level options available to me.

**Additional information you deem important (e.g. issue happens only occasionally):**
This appears to have been fixed at one point, but that fix was somehow lost: https://github.com/docker/docker/issues/10034

**Output of `docker version`:**
```
Client:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   23cf638
 Built:        Thu Aug 18 17:52:38 2016
 OS/Arch:      linux/amd64

Server:
 Version:      1.13.0-dev
 API version:  1.25
 Go version:   go1.7.1
 Git commit:   c2decbe
 Built:        Tue Sep 13 15:12:54 2016
 OS/Arch:      windows/amd64
```

**Output of `docker info`:**

```
Containers: 0
 Running: 0
 Paused: 0
 Stopped: 0
Images: 0
Server Version: 1.13.0-dev
Storage Driver: windowsfilter
 Windows:
Logging Driver: json-file
Plugins:
 Volume: local
 Network: nat null overlay
Swarm: inactive
Security Options:
Kernel Version: 10.0 14393 (14393.187.amd64fre.rs1_release_inmarket.160906-1818)
Operating System: Windows 10 Enterprise N
OSType: windows
Architecture: x86_64
CPUs: 8
Total Memory: 15.92 GiB
Name: Micah-Laptop
ID: ARD7:3DLJ:JRRU:M2BX:4PP3:I4Y5:FGBS:SOXQ:6A6R:YK5S:QM3J:QP3Y
Docker Root Dir: C:\ProgramData\Docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Insecure Registries:
 127.0.0.0/8
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
Same problem when using Windows CLI.

**Comment by thaJeztah** [comment]
Looks like it was removed in https://github.com/docker/docker/pull/10547, to keep the output under 80 chars wide. However, I see other commands output being wider, so let me open a PR to add it back (we can discuss on the PR)

**Comment by justincormack** [comment]
Does the `log-level` command do anything on the client? This is not discussed in the docs, and it does not appear to do anything...

**Comment by thaJeztah** [comment]
IIRC `docker --debug some-command` is the same as `docker --log-level=debug some-command`, but perhaps that's no longer the case with the split of daemon/client

**Comment by thaJeztah** [comment]
@justincormack looks like it's used https://github.com/docker/docker/blob/b248de7e332b6e67b08a8981f68060e6ae629ccf/cli/flags/common.go#L106-L117, https://github.com/docker/docker/blob/b248de7e332b6e67b08a8981f68060e6ae629ccf/cmd/docker/docker.go#L101-L111

But I'm not entirely sure in which cases it's being called; https://github.com/docker/docker/blob/b248de7e332b6e67b08a8981f68060e6ae629ccf/cmd/docker/docker.go#L42

ping @dnephin any idea? Should we move this to the `daemon` / `dockerd` command?

**Comment by dnephin** [comment]
I think there may be a few random places that use the logger in the client, but I don't recall exactly where it's called.

**Comment by justincormack** [comment]
@thaJeztah that looks like it is set up, but I am not sure it is used. I don't think it makes any sense in the client.

**Comment by BretFisher** [comment]
TL;DR answer: `--log-level` on dockerd can be one of `debug, info, warn, error, fatal`
